safe_control_gym/envs/gym_pybullet_drones/quadrotor.py

```diff
@@ -883,9 +904,16 @@ def _get_reset_info(self):
         '''
         info = {}
         info['symbolic_model'] = self.symbolic
+        # TODO: reverse or delete commented out code once decided how to fix prior
+        # info['physical_parameters'] = {
+        #     'quadrotor_mass': self.MASS,
+        #     'quadrotor_iyy_inertia': self.J[1, 1]
+        # }
         info['physical_parameters'] = {
```

Comment:
I'm not sure why this is logged in the info_dict, don't see any current controllers using it as well. maybe it's legacy code and we can safely remove it.

Reply:
the prior values can be fetched from the prior model. so here the pysical ones are the actual params used in simulation

Reply:
I guess there should be a 'safe' way to retrieve the true inertial parameters other than directly accessing the env (which Jacopo doesnt seem to like) so I guess this makes sense. can get it from info.

Reply:
actually i think we can keep this, say if we ever wanna test how domain randomization + prior works, then since each episode will have a differet set of params, it's useful to be able to access them in the info dict

Reply:
yah. if u agree then i will remove the commented/legacy part then

Reply:
yeah i think that makes sense, i hate lingering comments lmao feels cleaner to just delete legacy stuff so yeah leggo